README.rst

```diff
@@ -118,8 +113,8 @@ Meta
 .. |docs| image:: https://readthedocs.org/projects/pyobis/badge/?version=latest
    :target: http://pyobis.readthedocs.org/en/latest/?badge=latest
 
-.. |tests| image:: https://github.com/iobis/pyobis/actions/workflows/tests.yml/badge.svg
+.. |tests| image:: https://github.com/ayushanand18/pyobis/actions/workflows/tests.yml/badge.svg
```

Comment:
Please point all status check to the upstream repo and not your fork.

Reply:
Thank you @ocefpaf for the review! I am sorry I just missed updating this, I was testing whether the badge works or not but forgot to update it later. I'll make the necessary changes.